proto/remote_execution.proto

```diff
@@ -1753,6 +1763,12 @@ message TreeToken {
   repeated SizedDirectory sized_directories = 1;
 }
 
+/******************************************************************************/
+/*                                                                            */
+/*                BEGIN BUILDBUDDY-SPECIFIC PROTO DECLARATIONS                */
+/*                                                                            */
+/******************************************************************************/
```

Comment:
wdyt of this style of comment? it seemed to me like if we're trying to delineate blocks, it should be very visible?